.github/workflows/build.yml

```diff
@@ -262,13 +261,14 @@ jobs:
           path: ./src/vs-bicep/Bicep.VSLanguageServerClient.Vsix/bin/Release/vs-bicep.vsix
           if-no-files-found: error
 
-      - name: Run bicep in visual studio unit tests
-        uses: microsoft/vstest-action@v1.0.0
-        with:
-          testAssembly: Bicep.VSLanguageServerClient.UnitTests.dll
-          runInParallel: true
-          searchFolder: ./src/vs-bicep/Bicep.VSLanguageServerClient.UnitTests/bin/Release/net472/
-          ifNoFilesFound: error
+      # Temporarily disable: https://github.com/Azure/bicep/issues/14251
+      # - name: Run bicep in visual studio unit tests
+      #   uses: microsoft/vstest-action@v1.0.0
+      #   with:
+      #     testAssembly: Bicep.VSLanguageServerClient.UnitTests.dll
+      #     runInParallel: true
+      #     searchFolder: ./src/vs-bicep/Bicep.VSLanguageServerClient.UnitTests/bin/Release/net472/
+      #     ifNoFilesFound: error
```

Comment:
[nit]: Could you use an `if` here instead of commenting the step out? I haven't tried it out, but [step-level `if` properties should be supported](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif)

Reply:
What's the advantage?

Reply:
Went ahead and made the change, though.

Reply:
> What's the advantage?

None, I suppose. This is just a personal cognitive bias, but I always think commented out code was left there by mistake, whereas explicitly disabled code must have been intentional. The intention here was made clear by the comment, though.